src/main/java/org/wso2/carbon/hash/provider/pbkdf2/constant/Constants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.hash.provider.pbkdf2.constant;
+
+/**
+ * This class contains constants.
+ */
+public class Constants {
+
+    public static final String IDENTITY_HASH_PROVIDER_PBKDF2_ERROR_PREFIX = "UTA-";
```

Comment:
What is UTA?

Reply:
Also change as `PBKDF2_HASH_PROVIDER_ERROR_PREFIX`

Reply:
resolved the prefix variable name as `PBKDF2_HASH_PROVIDER_ERROR_PREFIX` and value as `PB2-`